eppo_core/src/events/event.rs

```diff
@@ -1,7 +1,7 @@
 use crate::timestamp::Timestamp;
 use serde::{Deserialize, Serialize};
 
-#[derive(Debug, Serialize, Deserialize, Clone)]
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
```

Comment:
> Clone, PartialEq

minor: when do we need to clone or compare events? Given that `payload` is arbitrarily nested JSON, cloning or comparing it are somewhat slow and we want to avoid that

Reply:
it's just for tests :)